BBallStatsTrack/src/com/example/bballstatstrack/models/Player.java

```diff
@@ -0,0 +1,265 @@
+package com.example.bballstatstrack.models;
+
+public class Player
+{
+    private static final String B_BALL_STAT_TRACK = "BBallStatTrack";
+
+    private int mNumber;
+
+    private String mFullName;
+
+    private int m2ptFGMiss = 0;
+
+    private int m2ptFGMade = 0;
+
+    private int m3ptFGMiss = 0;
+
+    private int m3ptFGMade = 0;
+
+    private int mFTMiss = 0;
+
+    private int mFTMade = 0;
+
+    private int mOffRebound = 0;
+
+    private int mDefRebound = 0;
+
+    private int mAssist = 0;
+
+    private int mTurnover = 0;
+
+    private int mSteal = 0;
+
+    private int mBlock = 0;
+
+    private int mFoul = 0;
+
+    private int mPlayingTimeSec = 0;
+
+    public Player( int id, String fullName )
+    {
+        setNumber( id );
+        setFullName( fullName );
+    }
+
+    public Player( int number, String fullName, int miss1pt, int miss2pt, int miss3pt, int made1pt, int made2pt,
```

Comment:
Builder patterns are best for situations such as this(Constructors having too many parameters). If you still have time you can implement it here.